cranelift/assembler-x64/meta/src/dsl.rs

```diff
@@ -32,7 +35,7 @@ pub fn inst(
         encoding,
         features: features.into(),
         has_trap: false,
-        custom_visit: false,
+        custom: CustomOperation::None.into(),
```

Comment:
```suggestion
        custom: Custom::default(),
```

We could `#[derive(Default)]` to create the empty set or we could create a `Custom::empty() -> Self` function if we want to be more clear.